packages/server/src/api/controllers/row/alias.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  QueryJson,
+  SearchFilters,
+  Table,
+  Row,
+  DatasourcePlusQueryResponse,
+} from "@budibase/types"
+import { getDatasourceAndQuery } from "../../../sdk/app/rows/utils"
+import { cloneDeep } from "lodash"
+
+class CharSequence {
+  static alphabet = "abcdefghijklmnopqrstuvwxyz"
+  counters: number[]
+
+  constructor() {
+    this.counters = [0]
+  }
+
+  get character() {
+    return this.counters.map(i => CharSequence.alphabet[i]).join("")
+  }
+
+  next() {
+    for (let i = this.counters.length - 1; i >= 0; i--) {
```

Comment:
Should this next not return the character, so we are sure we don't retrieve the same one twice?

Reply:
Good idea - safer that way!